### DISCUSSION TITLE
deposit transaction details not shown

### DISCUSSION THREAD

**Discussion by smzerehpoush** [firefly-iii/firefly-iii#7322] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

when i create a new deposit transaction and click on it, it faces with error. although i can edit the transactin
![image](https://user-images.githubusercontent.com/6152197/229378622-0533675a-7e57-4f6d-b846-e3b3f9a050ce.png)
![image](https://user-images.githubusercontent.com/6152197/229378674-9fd22b93-23ed-4b97-a946-d5c295995ccc.png)

### Debug information

SQLSTATE[42S02]: Base table or view not found: 1932 Table 'firefly.audit_log_entries' doesn't exist in engine (Connection: mysql, SQL: select * from `audit_log_entries` where `auditable_id` = 2286 and `auditable_type` = FireflyIII\Models\TransactionJournal and `audit_log_entries`.`deleted_at` is null)

### Expected behaviour

show transaction details

### Steps to reproduce

1. create a deposit transaction
2. click on it in list of transactions

**Comment by JC5** [comment]
Are you sure you followed all upgrade instructions?

https://docs.firefly-iii.org/firefly-iii/installation/upgrade/#straight-from-github

**Reply by smzerehpoush** [reply]
i've installed latest version using docker

**Reply by JC5** [reply]
OK. Once you have more info we can continue the search.
